public/app/plugins/datasource/tempo/dataquery.gen.ts

```diff
@@ -68,9 +68,9 @@ export const defaultTempoQuery: Partial<TempoQuery> = {
 };
 
 /**
- * search = Loki search, nativeSearch = Tempo search for backwards compatibility
+ * nativeSearch = Tempo search for backwards compatibility
```

Comment:
Is the comment correct? I think with the removal of the old search tab, we don't have any backward compatibility issues.

Reply:
Yeah I think you're right, will remove this comment in the PR that removes the old search.